randovania/data/json_data/prime3.txt

```diff
@@ -1830,9 +2159,16 @@ Asset id: 691790544662692104
       Trivial
   > Dock to Refinery Access
       Plasma Beam
+  > Dock to Gel Purification Site
+      After Gel Refinery Site Pipe
   > Dock to Main Lift
       Trivial
 
+> Event - Pipe; Heals? False
+  * Event Gel Refinery Site Pipe
+  > Dock to Gel Purification Site
+      Trivial
+
 ----------------
 Gel Purification Site
 Asset id: 14145904616390261937
```

Comment:
The doors don't connect to each other. Both should have requirements Grapple Swing or Screw Attack